src/css/cs/items/_kitchensink.scss

```diff
@@ -271,7 +271,7 @@
 
   // Custom font-widths, PS does not deliver classes to allow different font-sizes
   @for $i from 10 through 25 {
-    $width: $i * 1px;
+    $width: $i * 5px;
```

Comment:
@marcpechaitis was this change intended? For font's it should be fine to have 10,11,12px but not for margins / padding